docs/user-guide/jobs.md

```diff
@@ -212,9 +212,9 @@ The pattern names are also links to examples and more detailed description.
 
 |                            Pattern                                         | Single Job object | Fewer pods than work items? | Use app unmodified? |  Works in Kube 1.1? |
 | -------------------------------------------------------------------------- |:-----------------:|:---------------------------:|:-------------------:|:-------------------:|
-| [Job Template Expansion](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/job/expansions/README.md)          |                   |                             |          ✓          |          ✓          |
-| [Queue with Pod Per Work Item](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/job/work-queue-1/README.md)  |         ✓         |                             |      sometimes      |          ✓          |
-| [Queue with Variable Pod Count](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/job/work-queue-2/README.md) |                   |         ✓         |             ✓               |                     |          ✓          |
+| [Job Template Expansion](/docs/user-guide/job/expansions/README)           |                   |                             |          ✓          |          ✓          |
```

Comment:
Naturally, if the filenames change from README.md to index.md, this link URL would become /docs/user-guide/job/expansions/

Same for below